freezefrog/__init__.py

```diff
@@ -28,100 +63,103 @@ def __instancecheck__(self, obj):
 
 class FakeDateTime(with_metaclass(FakeDateTimeMeta, real_datetime)):
     """
-    A datetime replacement that lets you set utcnow() using set_utcnow().
+    A `datetime` mock class.
 
-    The clock starts ticking after calling set_utcnow().
+    Set the start datetime and the system timezone with `start(dt, tz)`. The
+    clock starts ticking after calling `start`.
     """
-    tz_delta = None
 
-    @classmethod
-    def utcnow(cls, *args, **kwargs):
-        if not hasattr(cls, 'dt'):
-            raise NotImplementedError(
-                'use {}.set_utcnow(datetime) first'.format(cls.__name__)
-            )
-        return cls._utcnow()
+    dt = None
+    tz = None
+    _start = None
 
     @classmethod
-    def set_utcnow(cls, dt):
+    def _initialize(cls, dt, tz, fold, tick):
         cls.dt = dt
+        cls.tz = tz
+        cls._start = time.monotonic() if tick else None
 
-    @classmethod
-    def set_tz_delta(cls, tz_delta):
-        cls.tz_delta = tz_delta
+        if is_pytz(tz):
+            cls.dt_in_utc = pytz.UTC.normalize(
+                tz.localize(dt, is_dst=get_pytz_dst_from_fold(dt, tz, fold))
+            )
+        else:
+            cls.dt_in_utc = dt.replace(tzinfo=tz, fold=fold).astimezone(
+                datetime.timezone.utc
+            )
 
     @classmethod
-    def _utcnow(cls):
-        if not hasattr(cls, '_start'):
-            cls._start = real_datetime.utcnow()
-        return (real_datetime.utcnow() - cls._start) + cls.dt
+    def _time_since_start(cls):
+        if cls._start is None:
+            return datetime.timedelta(seconds=0)
```

Comment:
Or just `datetime.timedelta()`

Reply:
Done in https://github.com/closeio/freezefrog/pull/13/commits/64cab51ee78993a56ab873c2883d273f5f4e21d4.